src/monstergenerator.cpp

```diff
@@ -147,7 +147,9 @@ const std::map<std::string, m_flag> flag_map = {
     { "DRIPS_NAPALM", MF_DRIPS_NAPALM },
     { "DRIPS_GASOLINE", MF_DRIPS_GASOLINE },
     { "ELECTRIC_FIELD", MF_ELECTRIC_FIELD },
-    { "LOUDMOVES", MF_LOUDMOVES }
+    { "LOUDMOVES", MF_LOUDMOVES },
+    { "DROPS_AMMO", MF_DROPS_AMMO } //NEW
```

Comment:
```suggestion
    { "DROPS_AMMO", MF_DROPS_AMMO }
```
It won't be new forever. and the comment doesn't convey any useful informtation.